webservices/common/models/dates.py

```diff
@@ -13,6 +13,10 @@ class ReportType(db.Model):
     report_type_full = db.Column('rpt_tp_desc', db.String, index=True, doc=docs.REPORT_TYPE)
 
 
+class DisclosureMixin(object):
+    __table__args = {"schema": "disclosure"}
```

Comment:
Oh interesting, nice!  We are going to have to do this for other models and tables, I wonder if this is a better way to approach this going forward rather than making the change in each model.  One benefit I see is that if the schema changes again for a set of tables we only have to adjust in the one spot, though that may also necessitate a change to the Mixin name too, which would have to propagate through the associated models again and thus not actually give any benefit there.

Still, I like this idea. :-)